tests/hypertext/htmlhint_test_files/.htmlhintrc

```diff
@@ -0,0 +1,4 @@
+{
+    "inline-script-disabled": true,
+    "inline-style-disabled": true
+}
```

Comment:
if you like ;) however sometimes test files shall contain such inconsistencies (not in your case though)